openedx/adg/lms/webinars/admin.py

```diff
@@ -67,6 +66,7 @@ class CancelledWebinarAdmin(WebinarAdminBase):
     """
     Model admin for cancelled webinar
     """
+    save_as = False
```

Comment:
We do not need the `save_as` here as it is False by default. We only need to override it to True in WebinarAdmin.